src/renderer/prefComponents/spellchecker/index.vue

```diff
@@ -37,16 +37,16 @@
       v-if="isOsx && !isHunspellSelected && spellcheckerEnabled"
       class="description"
     >
-      Please add required language dictionaries via "Language & Region" in your system preferences pane.
+      Additional languages may be added through "Language & Region" in your system preferences pane.
     </div>
     <div
       v-if="isWindows && !isHunspellSelected && spellcheckerEnabled"
       class="description"
     >
-      Please add required language dictionaries via "Language" in your "Time & language" settings. Add the langauge and download the "Basic typing" option.
+      Additional languages may be added via "Language" in your "Time & language" settings.
     </div>
     <div v-if="isHunspellSelected && spellcheckerEnabled">
-      <div class="description">List of available Hunspell dictionaries. Please add additional language dictionaries via the drop-down menu below.</div>
+      <div class="description">Installed Hunspell dictionaries. (Add additional language dictionaries below.)</div>
```

Comment:
```suggestion
      <div class="description">Installed Hunspell dictionaries.</div>
```

Reply:
Is this label referring to dictionaries "available" in the sense of being able to be installed, or being already installed?

Reply:
It's references to the installed dictionaries in the sense of being already installed. Here is a picture:

![mt-settings-spelling](https://user-images.githubusercontent.com/22716132/82739669-bb5b9d80-9d41-11ea-8eea-8b3025eaf096.png)

Then you can add additional dictionaries via the drop-down menu and button at the bottom of the picture.